treeherder/etl/job_loader.py

```diff
@@ -86,10 +105,12 @@ def validate_revision(self, repository, pulse_job):
             )
 
         if not Push.objects.filter(**filter_kwargs).exists():
+            (real_task_id, _) = task_and_retry_ids(pulse_job["taskId"])
             raise MissingPushException(
-                "No push found in {} for revision {}".format(
+                "No push found in {} for revision {} for task {}".format(
                     pulse_job["origin"]["project"],
-                    revision))
+                    revision,
+                    real_task_id))
```

Comment:
All of the changes on this file is so I can make use of `real_task_id` as part of the logging error.